### PR TITLE
Cosmos track 2 migration

### PR DESCRIPTION
**Pull request by g2vinay**
Moves Cosmos Track 2 code (V3 branch) as it is to 'sdk/cosmos/' location in central repo.

Follow up PR to:
1. rename sdk/cosmos/sdk to sdk/comos/azure-cosmos
2. Build Integration to central repo track 2 build.
3. Live + Emulator Tests Dev Ops pipeline integration.
4. Checkstyle + Spotbugs cleanup.

### CODE REVIEW

File: sdk/cosmos/benchmark/pom.xml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<!--
```
Comment (weshaggard) [inline sdk/cosmos/benchmark/pom.xml]: The folder here should be `sdk/cosmos/microsoft-azure-cosmos-parent` assuming this is track 1 based on the group id being `com.microsoft.azure`. @JonathanGiles is that correct?

File: sdk/cosmos/examples/pom.xml [hunk 2]
```diff
@@ -0,0 +1,159 @@
+<!--
```
Comment (weshaggard) [inline sdk/cosmos/examples/pom.xml]: For similar reasons as already stated this folder should be `sdk/cosmos/microsoft-azure-cosmos-examples`

File: sdk/cosmos/sdk/pom.xml [hunk 3]
```diff
@@ -0,0 +1,280 @@
+<!--
```
Comment (weshaggard) [inline sdk/cosmos/sdk/pom.xml]: Folder should be `sdk/cosmos/microsoft-azure-cosmos`

File: sdk/cosmos/benchmark/pom.xml (hunk 1 above)
Comment (weshaggard) [reply sdk/cosmos/benchmark/pom.xml]: fyi @mitchdenny

**Comment by weshaggard** [comment]
Looks like you need to update docsettings to fix the readme errors and I left a few comments about directory names otherwise this looks reasonable for an initial port. We will of course want to more integrate it into the repo as we start to do work on it and clean-up some things. It would be good to file an issue (or set of issues) to track that work.

File: sdk/cosmos/sdk/pom.xml (hunk 3 above)
Comment (g2vinay) [reply sdk/cosmos/sdk/pom.xml]: will cover it in follow up PR too.

File: sdk/cosmos/examples/pom.xml (hunk 2 above)
Comment (g2vinay) [reply sdk/cosmos/examples/pom.xml]: will cover it in follow up PR too.

File: sdk/cosmos/benchmark/pom.xml (hunk 1 above)
Comment (g2vinay) [reply sdk/cosmos/benchmark/pom.xml]: will do the folder renames in the follow up PR, which will wire up the builds to central parent pom, add Emulator CI +  live tests CI.

Comment (g2vinay) [reply sdk/cosmos/benchmark/pom.xml]: Just wondering, why is benchmark folder to be renamed to microsoft-azure-cosmos-parent

Comment (mitchdenny) [reply sdk/cosmos/benchmark/pom.xml]: Generally speaking we want the folder names containing the sources for a package to match the artifact name.

**Pull request merged**